common/modules/gcp-stackdriver-lbm/main.tf

```diff
@@ -0,0 +1,102 @@
+terraform {
```

Comment:
Since this is a new module, what about implementing these with native Terraform (I think we established the log-based metrics do work - https://github.com/terraform-providers/terraform-provider-google/issues/3698) instead of duplicating the whole `gcp-stackdriver-lbm/main.tf` module?

Reply:
Sounds a great idea, I'll address this in a separate ticket once this is merged.